website/frontend/templates/docs/tags.html

```diff
@@ -325,6 +325,13 @@ <h1 id="basic_variables">Basic Variables</h1>
            >Release Language</a> as per <a rel="nofollow" class="external text" href="http://www.sil.org/iso639-3">ISO 639-3</a> (since
             Picard 0.10)</td>
         </tr>
+        <tr>
+          <td>_releaselcountries</td>
```

Comment:
Is `_releaselcountries` correct??

Reply:
Yes

Reply:
ah, you mean there is a typo in _releasecountries. Thought you were asking about the underscore.